bolt/src/main/java/com/slack/api/bolt/AppConfig.java

```diff
@@ -339,6 +339,14 @@ public void setOauthRedirectUriPath(String oauthRedirectUriPath) {
 
     // --------------------------
 
+    /**
+     * Enables the state parameter in the OAuth flow. Enabling the validation is highly recommended
```

Comment:
Any suggestions about this sentence?

Reply:
How about "Enables validation of the state parameter in the OAuth flow. It is highly recommended to enable this validation for better security. A valid exception is when Enterprise Grid Org admins install apps from the app management page." ?